dll/win32/msi/dialog.c

```diff
@@ -3159,8 +3159,22 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
          * if a width is invalid, all remaining columns are hidden
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
+#ifdef __REACTOS__
+            if ((count == 0) && !(wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 )))
```

Comment:
```suggestion
            if ((count == 0) && (!wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 )))
```

Please review your own code before submitting it as a PR.....